mythril/analysis/symbolic.py

```diff
@@ -21,6 +22,10 @@ def __init__(self, contract, address, strategy, dynloader=None, max_depth=22,
             s_strategy = DepthFirstSearchStrategy
```

Comment:
What do you think about the following:
Modules in laser.strategy all define some metadata and are dynamically loaded vs doing it hardcoded here

Reply:
This should be a separate pr ofc

Reply:
:+1: